app/components/course-search-result.js

```diff
@@ -0,0 +1,18 @@
+import Component from '@ember/component';
+import { computed } from '@ember/object';
+import { reads } from '@ember/object/computed';
+
+export default Component.extend({
+  classNames: ['course-search-result', 'results-row'],
+  tagName: 'li',
+
+  course: null,
+  showMore: false,
+
+  courseTags: reads('course.matchedIn'),
+  sessions: reads('course.sessions'),
```

Comment:
Remove this property and just call the `computedSessions` property `sessions`.

Reply:
👍